### PR TITLE
Fix Issue #1308 not iterable

### PR DESCRIPTION
**Pull request by bissenbay**
Fixes: https://github.com/thoth-station/storages/issues/1308

### CODE REVIEW

File: thoth/storages/graph/postgres.py [hunk 1]
```diff
@@ -4889,43 +4889,45 @@ def sync_adviser_result(self, document: dict) -> None:
             )
 
             # Output stacks - advised stacks
-            for idx, result in enumerate(document["result"]["report"]):
```
Comment (fridex) [inline thoth/storages/graph/postgres.py]: Is this the following change?
```python
            for idx, result in enumerate(document["result"].get("report", [])):	
```

**Comment by goern** [comment]
recheck

File: thoth/storages/graph/postgres.py (hunk 1 above)
Comment (bissenbay) [reply thoth/storages/graph/postgres.py]: Thanks @fridex , this is way better

**Comment by goern** [comment]
recheck

**Pull request merged**